python/cudf/cudf/_lib/pylibcudf/libcudf/io/csv.pxd

```diff
@@ -95,6 +96,7 @@ cdef extern from "cudf/io/csv.hpp" \
         void set_quoting(cudf_io_types.quote_style style) except +
         void set_quotechar(char val) except +
         void set_doublequote(bool val) except +
+        void set_detect_whitespace_around_quotes(bool val) except +
```

Comment:
*question*: Do we want to plumb this option through the json reader? Or is it better to do so in a followup?

Reply:
I think a followup is fine.